### DISCUSSION TITLE
British Film Institute

### DISCUSSION THREAD

**Discussion by MartinEPhillips** [aclap-dev/video-downloadhelper#343] [Help and Q&A]
### Your question

Hi! Using Win 11 + up to date Firefox. I can download videos from the BFI (I have a subscription) using DVH  but they replay as gibberish. Currently trying with https://player.bfi.org.uk/subscription/film/watch-the-beast-1975-online

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.2.8 (stable)
Browser locale: en-GB
Production build
Built on 2024-03-18
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

**Comment by MartinEPhillips** [comment]
Update: I can download and view "free for all" videos, but not ones that I can only view with my subscription. I guess this is reasonable from the POV of the BFI, just wondering if there's a way around it.

**Reply by Mattmjs** [reply]
Gibberish you say, sounds like some content is protected with DRM. Do you see this icon, that would confirm it.
No way around that.

![DRM icon](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/0704dd9e-1045-4204-a2a2-bae7acd1bb68)
